File: Mathematics/printThePattern.cpp
void printPat(int n)
{
    for(int init = 0; init < n; init++){
        for(int i = n; i >= 1; i--){
            for(int j = n-init; j >= 1; j--){
                cout<<i<<" ";
            }
        }
        cout<<"$";
    }
}

File: Mathematics/printThePattern.py
def printPat(n):
    #Code here
    for i in range(n):
        # print(i)
        for j in range (n, 0, -1):
            for k in range(n-i, 0, -1):
                print(j, end =" ")
        print("$", end ="")
